TYPES.adoc

```diff
@@ -0,0 +1,94 @@
+= Content Types
+
+This is meant to resolve
+link:https://issues.jenkins-ci.org/browse/WEBSITE-21[WEBSITE-21] and define
+some of the content (read: data) types to be used for the `jenkins.io` site.
+
+
+== Timeline
+
+These content types can be thought of as a timeline of related pieces of
+content. Much like a blog or twitter feed, these pieces of content can be
+strung together on a chronological timeline.
+
+
+=== Blog Post
+
+**Template**: post
+
+**Data:**
+
+* `title`
+* `tags`
+* `body`
```

Comment:
keywords == tags in my book. summary/excerpt is something I'm on the fence about since Drupal and most other content generators use the first part of the blog post as the lead-in/summary. That seems to work out rather well.